front/src/applications/operationalStudies/components/Scenario/TimetableManageTrainSchedule.tsx

```diff
@@ -6,17 +6,20 @@ import { updateTrainScheduleIDsToModify } from 'reducers/osrdconf';
 import DotsLoader from 'common/DotsLoader/DotsLoader';
 import { useDispatch, useSelector } from 'react-redux';
 import { getTrainScheduleIDsToModify } from 'reducers/osrdconf/selectors';
+import { Infra } from 'common/api/osrdEditoastApi';
 import submitConfAddTrainSchedules from '../ManageTrainSchedule/helpers/submitConfAddTrainSchedules';
 import submitConfUpdateTrainSchedules from '../ManageTrainSchedule/helpers/submitConfUpdateTrainSchedules';
 
 type Props = {
   displayTrainScheduleManagement: string;
   setDisplayTrainScheduleManagement: (type: string) => void;
+  infra?: Infra;
```

Comment:
InfraState is enough here :) 
```suggestion
  infraState?: Infra['state'];
```

Reply:
done :)